### DISCUSSION TITLE
Export to Excel

### DISCUSSION THREAD

**Discussion by tycariadus** [moneymanagerex/moneymanagerex#4862] [Q&A] [answered]
In about the last ten versions or so I've found that the export to excel function only creates a blank excel worksheet, with no data coming through.  What am I missing?!  Thanks!

Simon

**Comment by vomikan** [comment, marked as answer]
You can select transactions in any account then past it to an Excel sheet.

![image](https://user-images.githubusercontent.com/6836805/180645772-eb51b119-8d0f-41e0-8312-909d9ab7724a.png)

Or export some account to XML format then open it in Excel.

**Comment by vomikan** [comment]
Also you can export to html then open it in Excel

![image](https://user-images.githubusercontent.com/6836805/180645926-1d42f37b-ed42-4585-a28b-c13780982600.png)

**Comment by vomikan** [comment]
You're probably talking about exporting a report to a spreadsheet. This is the context menu of the Internet Explorer. And it really doesn't work.
![image](https://user-images.githubusercontent.com/6836805/180646414-fd2451a4-1b63-44b4-b5c6-5054925553fc.png)

**Comment by tycariadus** [comment]
Hi , yes I was thinking of the exporting of a transaction report which used
to work but now you just get a blank excel spreadsheet.  This has to be a
bug?

 

Cheers

 

Simon
